apps/nextjs-website/react-components/components/Header/helpers/Header.Card.helpers.tsx

```diff
@@ -33,7 +33,7 @@ export default function ActionAreaCard({
         borderRadius: '15px',
       }}
     >
-      <CardActionArea>
+      <div>
```

Comment:
Why was `CardActionArea` removed in favor of a standard `div`?

Have you tested that we're not losing any functionality?

Reply:
CardActionArea has been removed because it makes the card clickable, only the button contained in the card must be clickable